lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.hvac_systems.rb

```diff
@@ -0,0 +1,245 @@
+class ASHRAE901PRM < Standard
+  DESIGN_CHILLED_WATER_TEMPERATURE = 44 # Loop design chilled water temperature (F)
```

Comment:
really don't like the use of global variables here.  I recommend the approach used in [Prototype.hvac_systems.rb](https://github.com/NREL/openstudio-standards/blob/master/lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb#7-L22L7).

Reply:
moved to chw_sizing_control method